about.html

```diff
@@ -0,0 +1,116 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Lato:wght@300;400;700;900&display=swap" 
+  rel="stylesheet">
+  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.3/css/all.css" 
+  integrity="sha384-SZXxX4whJ79/gErwcOYf+zWLeJdY/qpuqC4cAa9rOGUstPomtqpuNWT9wdPEn2fk" 
+  crossorigin="anonymous">
+  <link rel="stylesheet" href="style.css">
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet" 
+  integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" 
+  crossorigin="anonymous">
+  <title>About Psch Conference</title>
+</head>
+<body>
+  <main>
+    <header class="menuBar">
+      <div class="topnav" id="myTopnav">
+        <img class="img-responsive logo" src="images/psych-logo.png" alt="Psych Conference Logo">
+        <a href="#project">Project</a>
+        <a href="about.html">About</a>
+        <a href="index.html">Home</a>
+        <a href="javascript:void(0);" class="icon" id="menuToggle">
+          <i class="fa fa-bars"></i>
+        </a>
+      </div>
+    </header>
+    <div>
+      <section class="more-info">
+        <div>
+          <h1>
+            PSYCH AND MENTAL HEALTH | SPOKEN WORDS CONFERENCE 2021
+          </h1>
+          <p class="more-info-detail"> This is an bi-annual Psychology Conference that 
+            brings togther prominent Psychologists in the world, to share knowledge,
+            tackle problems and seek solutions to impending dangers of mental health.
+            This year's conference would be in a conducive environment of Sheraton
+            Hotels, Abuja, Nigeria. It would hold for 3days, from 17th - 20th of October, 2021.
+            For this year, we'll be expecting a host of professional in the field of
+            Mental Health and there would discussion time, where we socialize and learn.
+          </p>
+        </div>
+        <div class="email-us">
+          <p>For more information on this event, <br /> please contact the helpdesk!</p>
+          <a href="mailto:nnaemekasilver@gmail.com">nnaemekasilver@gmail.com</a>
+        </div>
+      </section>
+    </div>
+      <section>
```

Comment:
- [ ] The middle section of the `About` page seems to be missing. This section should contain a centered column that includes a section header, a divider, a paragraph of text, and an image or large logo.

![original](https://user-images.githubusercontent.com/26856491/124120123-4a6dd580-da6b-11eb-9a9d-a156fdeb4ea3.png)
**_The middle section of the about page from the original design_**

Reply:
Nicely done! 💯